contracts/freelancer_dispute_resolver.py

```
@@ -0,0 +1,156 @@
+        self.client = gl.message.sender_account
+        self.freelancer = Address(freelancer_address)
+        self.job_description = job_description
+        self.deliverables_url = ""
+        self.client_evidence = ""
+        self.freelancer_evidence = ""
+        self.dispute_raised = False
+        self.resolved = False
+        self.verdict = ""
+        self.verdict_reasoning = ""
+
+    @gl.public.write
```

Ruff (0.15.1)

[error] `gl` may be undefined, or defined from star imports

(F405)